charts/nginx-ingress-controller/values.yaml

```diff
@@ -13,59 +13,62 @@
 # limitations under the License.
 
 nginx:
-  hostNetwork: false
-  asDaemonSet: false
-  replicas: 3
-  image:
-    repository: k8s.gcr.io/ingress-nginx/controller
-    tag: v1.0.2
-  config: {}
-#   load-balance: "least_conn"
-  extraArgs: []
-  resources:
-    requests:
-      cpu: 50m
-      memory: 128Mi
-    limits:
-      cpu: 500m
-      memory: 512Mi
-  nodeSelector: {}
-  affinity:
-    nodeAffinity:
-      preferredDuringSchedulingIgnoredDuringExecution:
-      - weight: 100
-        preference:
-          matchExpressions:
-          - key: kubermatic.io/type
-            operator: In
-            values:
-            - stable
-  tolerations:
-  - key: only_critical
-    operator: Equal
-    value: "true"
-    effect: NoSchedule
+  fullnameOverride: nginx-ingress
 
-  # set this to true to automatically add these tolerations
-  # to make nginx run on master nodes:
-  #   - { key: dedicated, operator: Equal, value: master, effect: NoSchedule }
-  #   - { key: node-role.kubernetes.io/master, effect: NoSchedule }
-  ignoreMasterTaint: false
+  # values for the ingress-nginx chart start below
+  # reference: https://github.com/kubernetes/ingress-nginx/blob/main/charts/ingress-nginx/values.yaml
+  controller:
+    hostNetwork: false
+    replicas: 3
```

Comment:
@wurbanski `controller.replicas` doesnt seem to exist in https://github.com/kubernetes/ingress-nginx/blob/main/charts/ingress-nginx/values.yaml, it's `controller.replicaCount`. The nginx-ingress-controller Deployment on dev has only one replica right now.